studiopress/genesis-simple-menus
# Remove Gunt and add POT generation via wp-cli

**Summary of change:**
This PR replaces Grunt and introduces the POT generation via wp-cli

**This PR has been:**

**Have the changes in this PR been added to the documentation for this project?**
Does not apply
(if No, please create and link to the issue that identifies the need for documentation)

**How to test:**
- Download this branch;
- Run `composer install` and `composer makepot`;
- Check if the `.pot` file has been properly generated.

**See:** #xxx

**Fixes:** #xxx

**Suggested Changelog Entry:**
- Introduces POT generation via wp-cli.

_Note:_ Anything else that should be noted for this PR
